drivers/dac/ad3552r/ad3552r.c

```diff
@@ -1417,6 +1442,12 @@ int32_t ad3552r_init(struct ad3552r_desc **desc,
 	}
 	ldesc->chip_id = param->chip_id;
 	ldesc->is_simultaneous = param->is_simultaneous;
+	if (ldesc->chip_id == AD3542R_ID ||  ldesc->chip_id == AD3541R_ID) {
```

Comment:
nit: drop brackets if only one expression is guarded. 
Also you swap to a switch case statement. It's easier to extend if new chips are added.